kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -201,16 +203,19 @@
           sug.toLowerCase().startsWith(this.username.toLowerCase())
         );
       },
-      usernameIsInvalidText() {
+      usernameErrorText() {
         if (this.usernameBlurred || this.formSubmitted) {
           if (this.username === '') {
             return this.$tr('required');
           }
         }
+        if (this.username && !validateUsername(this.username)) {
+          return this.$tr('usernameNotAlphaNumUnderscore');
```

Comment:
I wonder if we should do one of:

* rename `validateUsername` to `validateUsernameIsAlphanumeric`
* return `this.$tr('usernameNotAlphaNumUnderscore')` from `validateUsername`

Currently this logic assumes that `validateUsername` doesn't, for example, check max length